CHANGELOG.md

```diff
@@ -6,16 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-There are a number of backwards incompatible changes which are on the `master` branch
-waiting to be released. These are:
+# Version 4.0.0-alpha.1
 
-- Add Mapping::contains(key) and Mapping::insert_return_size(key, val) ‒ [#1224](https://github.com/paritytech/ink/pull/1224)
-- Optimise deny_payment. Use everywhere semantic of deny ‒ [#1267](https://github.com/paritytech/ink/pull/1267)
-- Implement ecdsa_to_eth_address() and remove eth_compatibility crate ‒ [#1233](https://github.com/paritytech/ink/pull/1233)
+## Breaking Changes
+This release contains a few breaking changes. These are indicated with the :x: emoji.
+Most of these were intitially introduced in `v3.1.0` and `v3.2.0` releases but
+compatibility was restored in `v3.3.0`.
```

Comment:
Could you add something on how to migrate and what the implications are? So along the lines of https://github.com/paritytech/ink/blob/master/CHANGELOG.md#breaking-changes.